src/isPalindrome.js

```diff
@@ -0,0 +1,5 @@
+export default function isPalindrome(input){
+  let firstString = input.toUpperCase().split('').filter((n)=>n.match(/[A-Za-z0-9]/)).join('')
```

Comment:
Make variable names more explicit. Something like `forwardsString` and `reversedString`? Also don't use `n` as the value of filter. Call it `character` or something describing what it is.